<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowNotificationDlg</class>
 <widget class="QDialog" name="ShowNotificationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="control::MwcLabelSmall" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>69</y>
     <width>630</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="timeText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>630</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="levelText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>165</y>
     <width>630</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>252</y>
     <width>630</width>
     <height>166</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="control::MwcLabelSmall" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>144</y>
     <width>630</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="pushButton">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>447</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="control::MwcLabelLarge" name="label">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>21</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Notification message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="control::MwcLabelSmall" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>228</y>
     <width>630</width>
     <height>19</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
